cookbooks/tian_parameterization_kinematic_slab/uncoupled-two-phase-tian-parameterization-kinematic-slab.prm

```diff
@@ -0,0 +1,27 @@
+# This input file creates a model with the same setup as the model generated by
+# coupled-two-phase-tian-parameterization-kinematic-slab.prm, with the only difference
+# being that the solid-fluid reactions in this parameter file are not fully coupled.
+# Instead, the fluid is advected only with the Darcy velocity. See
```

Comment:
I don't understand this input file. Don't you have to make the porosity a Darcy field? Where does that happen?